agent-inject/agent/annotations.go

```diff
@@ -37,6 +37,14 @@ const (
 	// secret-volume-path annotation.
 	AnnotationAgentInjectFile = "vault.hashicorp.com/agent-inject-file"
 
+	// AnnotationAgentInjectFilePermission is the key of the annotation that contains the
+	// permission of the file to create on disk. The name of the
+	// secret is the string after "vault.hashicorp.com/agent-inject-file-permission-", and
+	// should map to the same unique value provided in
+	// "vault.hashicorp.com/agent-inject-secret-". The value is the value of the permission, for
+	// example "0644"
+	AnnotationAgentInjectFilePermission = "vault.hashicorp.com/agent-inject-file-permission"
```

Comment:
Keeping in mind the 63 character limit for annotations, what do you think about making this shorter? Something like `vault.hashicorp.com/agent-inject-file-perms` or even just `vault.hashicorp.com/agent-inject-perms`?

Reply:
Good point. I've renamed it to your second, shorter suggestion.